### ISSUE TITLE
[x86-core2] "CPU ISA level is lower than required" issue with glibc 2.33

### ISSUE THREAD

**Issue by RomainNaour** [bootlin/toolchains-builder#40]
Hello,

When testing glibc 2.33 with x86-core2 toolchain, the system doesn't boot due to a kernel panic:

```
Run /sbin/init as init process
/lib/libc.so.6: CPU ISA level is lower than required
Kernel panic - not syncing: Attempted to kill init! exitcode=0x00007f00
```
This is due to a new isa level check added by this commit:
["x86: Support GNU_PROPERTY_X86_ISA_1_V[234] marker [BZ #26717]"](https://sourceware.org/git/?p=glibc.git;a=commit;h=ecce11aa0752735c4fd730da6e7c9e0b98e12fb8)

We get this issue even by using the -cpu core2duo on the Qemu command line (core2duo should match the defconfig option BR2_x86_core2)...
The boot succeed as soon as we use a newer microarchitecture like Broadwell, Nehalem, SandyBridge, Skylake on the Qemu command line.

Any ideas?

**Comment by RomainNaour** [comment]
The issue is already reported by:
https://sourceware.org/pipermail/libc-alpha/2021-February/122268.html

I added another report about our defconfig:
https://sourceware.org/pipermail/libc-alpha/2021-February/122920.html

**Comment by tpetazzoni** [comment]
Did you get to any conclusion about this ?

**Comment by RomainNaour** [comment]
Hello Thomas,

AFAIK, the issue should be fixed in the latest version of glibc 2.33 from the stable branch:

https://sourceware.org/git/?p=glibc.git;a=commit;h=ee9f98d9cac12e843ca59c6e4d4b225f58a66727

**Comment by RomainNaour** [comment]
I'm closing this issue since it's fixed in glibc 2.33 stable branch.
